src/app/(root)/create/page.tsx

```diff
@@ -19,6 +19,17 @@ export default function Dashboard() {
                 { placeholder: "Symbol" },
                 { placeholder: "Token Address (existing ERC-20 token)" },
               ]}
+              dropdowns={[
+                {
+                  placeholder: "Select Chains",
+                  options: [
+                    { value: "1", label: "Ethereum Mainnet" },
```

Comment:
Let's just say "Ethereum" instead of "Ethereum Mainnet".

Reply:
Sure, I have made the change.